modules/planning/common/reference_line_info.h

```diff
@@ -48,6 +48,9 @@ class ReferenceLineInfo {
       const common::TrajectoryPoint& init_adc_point,
       const ReferenceLineSmootherConfig& smoother_config);
 
+  ReferenceLineInfo(const ReferenceLineInfo&) = delete;
```

Comment:
Use DISALLOW_COPY_AND_ASSIGN macro. Also you need to do the same thing for PathDecision which owns an IndexedList.

Reply:
I will refactor IndexedList. It should  be a general container, not a container only for unique_ptr.